pkgs/sops-install-secrets/main.go

```diff
@@ -1141,7 +1147,8 @@ func writeTemplates(targetDir string, templates map[string]*template, keysGID in
 		if !userMode {
 			if err := os.Chown(fp, template.owner, template.group); err != nil {
 				return fmt.Errorf("cannot change owner/group of '%s' to %d/%d: %w", fp, template.owner, template.group, err)
-			} }
```

Comment:
This was changed by `gofmt`. This snuck in in https://github.com/Mic92/sops-nix/commit/aa5caa129bd96b7883fa5164dae7d91279ecb9d2#diff-5ebcd22450b1d2333c6ae4ac07ebca561b4f3ac2b6ea9e38d4c0049503775d49R1071, sorry!